test/api_test.go

```diff
@@ -165,23 +165,24 @@ func TestAPIcensusAndVote(t *testing.T) {
 	server.VochainAPP.AdvanceTestBlock()
 	waitUntilHeight(t, c, 3)
 
+	// Verify the vote
 	_, code = c.Request("GET", nil, "votes", "verify", election.ElectionID.String(), v.VoteID.String())
 	qt.Assert(t, code, qt.Equals, 200)
 
-	/*
-	   This test is disabled because the indexer is not properly set up in the test environment.
-	   Activate it when the indexer is properly working.
-
-	   resp, code = c.Request("GET", nil, "votes", v.VoteID.String())
-	   qt.Assert(t, code, qt.Equals, 200)
-	   v2 := &api.Vote{}
-	   err = json.Unmarshal(resp, v2)
-	   qt.Assert(t, err, qt.IsNil)
-	   qt.Assert(t, v2.VoteID.String(), qt.Equals, v.VoteID.String())
-	   qt.Assert(t, v2.BlockHeight, qt.Equals, uint32(3))
-	   qt.Assert(t, v2.TransactionIndex, qt.Equals, uint32(0))
-	   qt.Assert(t, v2.VoterID.String(), qt.Equals, voterKey.AddressString())
-	*/
+	//  This test is disabled because the indexer is not properly set up in the test environment.
```

Comment:
Can you please add the `TODO` keyword so we can later filter all TODO's ?

Reply:
Actually, I re-enabled this test, the comment was just a garbage.